npm/webpack-dev-server/src/plugin.ts

```diff
@@ -52,85 +53,98 @@ export default class CypressCTOptionsPlugin {
     this.devServerEvents = options.devServerEvents
   }
 
-  private pluginFunc = (context: CypressCTWebpackContext) => {
-    context._cypress = {
+  private addLoaderContext = (loaderContext: object, module: any) => {
```

Comment:
Seems the second arg, `module` isn't used?

Reply:
Typescript complains if I leave it out, because two arguments is the signature it expects for `loader.tap('CypressCTOptionsPlugin', callback)`.